src/utils/getTeams.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getTeams } from './getTeams';
+
+describe('getTeams', function () {
+  it('should return empty array if no teams were found', () => {
+    expect(getTeams('repo-does-not-exist', undefined, 'getsentry')).toEqual([]);
+  });
+
+  it('should return array with one team if repo does not have routing', () => {
+    expect(getTeams('test-ttt-simple', undefined, 'getsentry')).toEqual(['team-ospo']);
```

Comment:
1. It's a bit difficult to figure out what's being tested when looking at the test based on name alone. I guess this is a repo with no routing, but I'm not 100% sure?

Reply:
Ah nm, it's in product-owners.yml. May be worthwhile to say somewhere around here that that's where you go to look for the data under test.